new-backend/app/modules/langgraph_builder.py

```
@@ -0,0 +1,94 @@
+    judge,
+    store_and_send,
+    error_handler
+    )
+
+
+def build_langgraph():
+    graph = StateGraph()
+
+    graph.add_node(
+                    "sentiment_analysis",
+                    sentiment.run_sentiment
```

Pylint (3.3.7)

[error] Module 'app.modules.langgraph_nodes.sentiment' has no 'run_sentiment' member

(E1101)